### PR TITLE
feat: dispute analytics

### PR DESCRIPTION
**Pull request by sagarnaikjuspay**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates
- [ ] Documentation
- [ ] CI/CD

## Checklist

- [x] I ran `npm run re:build`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

File: src/entryPoints/SidebarValues.res [hunk 1]
```diff
@@ -189,6 +189,13 @@ let paymentAnalytcis = SubLevelLink({
   searchOptions: [("View analytics", "")],
 })
 
+let disputeAnalytcis = SubLevelLink({
+  name: "Disputes",
+  link: `/analytics-disputes`,
+  access: Access,
+  searchOptions: [("View analytics", "")],
```
Comment (Riddhiagrawal001) [inline src/entryPoints/SidebarValues.res:196]: Search option could be `View disputes analytics` or `View dispute graph`

File: src/entryPoints/SidebarValues.res [hunk 2]
```diff
@@ -189,6 +189,13 @@ let paymentAnalytcis = SubLevelLink({
   searchOptions: [("View analytics", "")],
 })
 
+let disputeAnalytcis = SubLevelLink({
```
Comment (Riddhiagrawal001) [inline src/entryPoints/SidebarValues.res:192]: `disputeAnalytcis` typo

File: src/screens/Analytics/AnalyticsTypes.res [hunk 3]
```diff
@@ -117,6 +117,35 @@ type refundsSingleStateSeries = {
   refund_processed_amount: float,
 }
 
+type disputeColType =
```
Comment (Riddhiagrawal001) [inline src/screens/Analytics/AnalyticsTypes.res:120]: Possible to make different type definition file for disputes ?

File: src/screens/Analytics/DisputeAnalytics/DisputeAnalyticsEntity.res [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+let percentFormat = value => {
+  `${value->Float.toFixedWithPrecision(~digits=2)}%`
+}
+
+let distribution =
+  [
+    ("distributionFor", "dispute_error_message"->JSON.Encode.string),
+    ("distributionCardinality", "TOP_5"->JSON.Encode.string),
+  ]
+  ->Dict.fromArray
```
Comment (Riddhiagrawal001) [inline src/screens/Analytics/DisputeAnalytics/DisputeAnalyticsEntity.res:39]: LogicUtils.getJsonFromArrayOfJson

**Review by JeevaRamu0104 — APPROVED** [review]

**Pull request merged**